crimsobot/cogs/reactions.py

```diff
@@ -132,14 +137,13 @@ async def fact_inspect(self, ctx: commands.Context, fact_id: int) -> None:
 
         field_list = [
             ('Subject', fact_object.subject),
-            ('Body', fact_object.body),
+            ('Body', fact_object.body[:500] + '...' if len(fact_object.body) > 500 else fact_object.body),
             ('Added by', f'{fact_adder} ({fact_adder.id})'),
             ('Server', f'{guild.name} ({guild.id})'),
             (
                 'Added on',
                 '{d.year}-{d.month:02d}-{d.day:02d} {d.hour:02d}:{d.minute:02d}:{d.second:02d}'.format(
-                    d=fact_object.created_at
-                )
+                    d=fact_object.created_at)
```

Comment:
I changed my mind on this again, I don't know what to do I don't like either option oh god